EarthAsDMAnalyzer/plugins/EarthAsDMAnalyzer.cc

```diff
@@ -204,6 +207,20 @@ class EarthAsDMAnalyzer : public edm::one::EDAnalyzer<edm::one::SharedResources>
   std::vector<float> muon_dtSeg_eta_;
   std::vector<float> muon_dtSeg_phi_;
 
+  std::vector<float> Track_Reconsruction_Score_;
```

Comment:
```suggestion
  std::vector<float> muon_r2_;
```

 * this does not belong to the general track loop
 * you had a typo in reco
 * you calculate the r2 so why not call it r2?